app/src/main/java/comp3350/mealbuddy/business/AccessExercise.java

```diff
@@ -0,0 +1,29 @@
+package comp3350.mealbuddy.business;
+
+import comp3350.mealbuddy.application.Main;
+import comp3350.mealbuddy.application.Services;
+import comp3350.mealbuddy.objects.Exercise;
+import comp3350.mealbuddy.persistence.DataAccessStub;
+
+public class AccessExercise {
+    private DataAccessStub DAS;
```

Comment:
I feel like this would be better being an interface and then we can plug in the stub or the real thing but since braico didn't do that in his sample I agree we can just instantiate it.

Reply:
I feel like an interface makes sense too. I just wanted to stay true to the sample, I can adjust it if need be.

Reply:
No I agree with you.  Since the sample did it we definitly won't lose marks.  Once we start using SQL it may be very useful for testing though for di.